.github/workflows/slsa-builder-go.yml

```diff
@@ -7,16 +7,23 @@ env:
   GENERATED_BINARY: generated-go-binary
   #TODO: let caller upload ?
   REGISTRY: ghcr.io
+  BUILDER_URL: https://github.com/laurentsimon/slsa-on-github
+  BUILDER_FOLDER: builder
 
+###################################################################
+#                                                                 #
+#            Input and output argument definitions                #
+#                                                                 #
+###################################################################
 on:
   workflow_call:
     inputs:
       go-version:
         description: "The go version to use"
         required: true
         type: string
-      working-dir:
```

Comment:
maybe let's also add an input for upload location? 
we can parametrize the hard-coded ghcr.io/${{ github.repository }}

Reply:
Done. Untrusted input again ^^